<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogChordDefinition</class>
 <widget class="QDialog" name="DialogChordDefinition">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1107</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chord Definition</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_17">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_11">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QTabWidget" name="tabWidget">
           <property name="currentIndex">
            <number>1</number>
           </property>
           <property name="movable">
            <bool>true</bool>
           </property>
           <property name="tabBarAutoHide">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="tabNewChord">
            <attribute name="title">
             <string>Chord Definition</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_16">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_15">
               <item>
                <widget class="Neck" name="neck">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout">
                   <item>
                    <widget class="QLabel" name="labelChordName">
                     <property name="text">
                      <string>Chord name</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="labelTranslatedChordnName">
                     <property name="text">
                      <string>Translated chord name</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_12">
                   <item>
                    <widget class="QLineEdit" name="lineEditChord">
                     <property name="readOnly">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="lineEditTranslatedChord">
                     <property name="readOnly">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QListWidget" name="listWidgetChords">
                   <property name="maximumSize">
                    <size>
                     <width>150</width>
                     <height>60</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEditNotation">
                   <property name="readOnly">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEditName"/>
                 </item>
                 <item>
                  <widget class="QToolButton" name="toolButtonPlus">
                   <property name="font">
                    <font>
                     <pointsize>20</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>+</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_13">
                   <item>
                    <widget class="QLabel" name="labelInDataBase">
                     <property name="text">
                      <string>In database</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="labelApproved">
                     <property name="text">
                      <string>Approved</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_14">
                   <item>
                    <widget class="QLabel" name="labelFlagDatabase">
                     <property name="text">
                      <string/>
                     </property>
                     <property name="pixmap">
                      <pixmap resource="../Resources/resources.qrc">:/Image/Images/task-reject.png</pixmap>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="labelFlagApproved">
                     <property name="text">
                      <string/>
                     </property>
                     <property name="pixmap">
                      <pixmap resource="../Resources/resources.qrc">:/Image/Images/task-reject.png</pixmap>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_2">
            <attribute name="title">
             <string>Chord Management</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_12">
               <item>
                <widget class="QTableView" name="tableViewChordDefinition"/>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_8">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_11">
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_7">
                     <item>
                      <widget class="QLabel" name="labelSortON">
                       <property name="text">
                        <string>Sort on</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="labelFilter">
                       <property name="text">
                        <string>Filter</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_6">
                     <item>
                      <layout class="QHBoxLayout" name="horizontalLayout_9">
                       <item>
                        <widget class="QRadioButton" name="radioButtonChordName">
                         <property name="text">
                          <string>Cho&amp;rd </string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QRadioButton" name="radioButtonFret">
                         <property name="text">
                          <string>fret</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QRadioButton" name="radioButtonSortIndex">
                         <property name="text">
                          <string>Index</string>
                         </property>
                         <property name="checked">
                          <bool>true</bool>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                     <item>
                      <layout class="QHBoxLayout" name="horizontalLayout_10">
                       <item>
                        <widget class="QLineEdit" name="lineEditFilter"/>
                       </item>
                       <item>
                        <widget class="QToolButton" name="toolButtonClearFilter">
                         <property name="text">
                          <string>...</string>
                         </property>
                         <property name="icon">
                          <iconset resource="../Resources/resources.qrc">
                           <normaloff>:/Image/Images/cross.png</normaloff>:/Image/Images/cross.png</iconset>
                         </property>
                         <property name="autoRaise">
                          <bool>true</bool>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <spacer name="verticalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButtonInsertChord">
                   <property name="text">
                    <string>Insert Chord </string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="verticalSpacer">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_5">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_7">
                   <item>
                    <widget class="QLineEdit" name="lineEditNameIndex"/>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="lineEditNameTranaslatedIndex"/>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="lineEditValueIndex"/>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_8">
                   <item>
                    <spacer name="horizontalSpacer_3">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QPushButton" name="pushButtonDelete">
                     <property name="text">
                      <string>Delete</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="pushButtonModify">
                     <property name="text">
                      <string>Modifiy</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer_4">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="ChordDiagram" name="widget" native="true"/>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_3">
            <attribute name="title">
             <string>Approve</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_16">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_15">
               <item>
                <widget class="QTableView" name="tableViewNonApproved"/>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_10">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_13">
                   <item>
                    <widget class="QLineEdit" name="lineEditNameNoApproved"/>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="lineEditTranslatedNameNoApproved"/>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="lineEditValueNoApproved"/>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_14">
                   <item>
                    <spacer name="horizontalSpacer_5">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QPushButton" name="pushButtonDeleteApprove">
                     <property name="text">
                      <string>Delete</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="pushButtonApprove">
                     <property name="text">
                      <string>Approve</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer_6">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="ChordDiagram" name="widget_2" native="true"/>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab">
            <attribute name="title">
             <string>Chord Import</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QPlainTextEdit" name="plainTextEditImport"/>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <item>
                  <widget class="QLabel" name="label81">
                   <property name="text">
                    <string>Copy paste chords in form </string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>chord=relative position 6 ..position 1 </string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>C(III)=3 0 0 3 3 3 0  </string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_5">
                   <item>
                    <spacer name="horizontalSpacer_2">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QPushButton" name="pushButtonImport">
                     <property name="text">
                      <string>Import</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonCancel">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Neck</class>
   <extends>QGraphicsView</extends>
   <header>neck.h</header>
  </customwidget>
  <customwidget>
   <class>ChordDiagram</class>
   <extends>QWidget</extends>
   <header>chorddiagram.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
